windows/internal/cuda_install.bat

```diff
@@ -185,22 +185,15 @@ echo Installing CUDA toolkit...
 7z x %CUDA_SETUP_FILE% -o"%SRC_DIR%\temp_build\cuda"
 pushd "%SRC_DIR%\temp_build\cuda"
 
-rem Why find ":", if no process, the output is INFO: No tasks are running which match the specified criteria.
-tasklist /fi "imagename eq msiexec.exe" | find ":"
-if %ERRORLEVEL% NEQ 0 (
-    echo There's another installer running.
-)
-
-start /wait setup.exe -s %ARGS%
+sc query wususerv
+sc config wuauserv start= disabled
```

Comment:
cc @seemethere @malfet I'm not sure whether it will break something at the CircleCI side or not. Would you please open a ticket about it?

Reply:
Windows update would start again after `net stop wususerv`

Reply:
But you set it to disabled instead of stopping it here, right?

Reply:
Yes. So far, CI only pass as the update service is disabled.

Reply:
I feel like ideally they wouldn’t be re-using runners between runs so this shouldn’t be a big issue.